update doc of annotatePeakInBath

hukai916 opened the pull request:
Hi Jianhong,

I simply updated the Roxygen comments and recreated the corresponding Rd file.
Let me know if any questions.

hukai916 committed:
update doc

jianhong commented on R/annotatePeakInBatch.R:
```diff
@@ -1,37 +1,40 @@
-#' Obtain the distance to the nearest TSS, miRNA, and/or exon for a list of
-#' peaks
+#' Obtain the distance to the nearest TSS, miRNA, and/or exon for a list of peaks
```
the newline here is used to follow 80 characters per line. Do not need to change.

jianhong reviewed (changes requested):
Also need to add items in news and bump the version number.

hukai916 committed:
update documenation and .Rd; move NEWS to top-level

hukai916 commented:
Hi Jianhong,

I have updated the documentation to follow the 80-character rule; updated the .Rd file; and moved/updated the NEWS to the top-level in order to be able to use the "news()" function.
